diff --git a/prod/settings.py b/prod/settings.py
index a4e3a6f..4a2774a 100644
--- a/prod/settings.py
+++ b/prod/settings.py
@@ -2,10 +2,10 @@ import os
 
 ENVIRONMENT = os.getenv('ENVIRONMENT', 'development')
 
-DEBUG = False  #TODO: change to false for production
+DEBUG = True  #TODO: change to false for production
 BASE_DIR = os.path.dirname(os.path.dirname(os.path.abspath(__file__)))
-SECRET_KEY = os.getenv('SECRET_KEY')
-ALLOWED_HOSTS = ['.callidonequipment.com', '3.18.11.161', '.callidongroup.com']
+SECRET_KEY = 'dummy'  # for testing
+ALLOWED_HOSTS = ['localhost']
 
 INSTALLED_APPS = [
     'django.contrib.admin',
@@ -63,11 +63,9 @@ USE_TZ = True
  
 STATIC_URL = '/static/'
 STATICFILES_DIRS = [os.path.join(BASE_DIR, 'static_in_env')]
-# STATIC_ROOT = os.path.join(BASE_DIR, 'static_root')
-STATIC_ROOT = '/var/www/callidonequipment.com/static'
+STATIC_ROOT = os.path.join(BASE_DIR, 'static_root')
 MEDIA_URL = '/media/'
-# MEDIA_ROOT = os.path.join(BASE_DIR, 'media_root')
-MEDIA_ROOT = '/var/www/callidonequipment.com/media'
+MEDIA_ROOT = os.path.join(BASE_DIR, 'media_root')
 
 DATABASES = {
     "default": {
